certbot/certbot/_internal/plugins/manual.py

```diff
@@ -44,12 +44,22 @@ class Authenticator(common.Plugin):
         'remain after the current one, and $CERTBOT_ALL_DOMAINS contains a comma-separated '
         'list of all domains that are challenged for the current certificate.')
     _DNS_INSTRUCTIONS = """\
-Please deploy a DNS TXT record under the name
-{domain} with the following value:
+Please deploy a DNS TXT record with the following value:
 
 {validation}
 
-Before continuing, verify the record is deployed."""
+under the name:
+
+{domain}.
+"""
+    _DNS_VERIFY_INSTRUCTIONS = """
+Before continuing, verify the TXT record has been deployed. Depending on the DNS 
+provider, this may take some time, from a few seconds to multiple minutes. You can
+check if it has finished deploying with aid of online tools, such as the Google
+Admin Toolbox: https://toolbox.googleapps.com/apps/dig/#TXT/{domain}.
+Look for one or more bolded line(s) below the line ';ANSWER'. It should (also)
```

Comment:
What's the function of the `(also)` here? Do we need it? The message seems clear without it to me.

Reply:
There could be multiple TXT records present, multiple challenge records, TXT records for SPF/DMARC et cetera. But if you think the "one or more" before is enough, I can remove it.